tests/system/common.py

```diff
@@ -226,38 +226,21 @@ def systemctl_master(command='restart'):
     shakedown.run_command_on_master('sudo systemctl {} dcos-mesos-master'.format(command))
 
 
-def save_iptables(host, filename='iptables.rules'):
-    """ Saves iptables firewall rules such they can be restored
-    """
-
-    shakedown.run_command_on_agent(
-        host,
-        'if [ ! -e {} ] ; then sudo iptables-save > {} ; fi'.format(filename, filename))
-
-
-def restore_iptables(host, filename='iptables.rules'):
-    """ Reconnect a previously partitioned node to the network
-        :param hostname: host or IP of the machine to partition from the cluster
-    """
-
+def block_iptable_rules(host, port_number, sleep_seconds, input=True, output=True):
```

Comment:
Could you add some docs? This implementation is not trivial.

Some nit: How about `block_iptable_rules_for_seconds`.